container_api/src/container/base.rs

```diff
@@ -142,11 +195,12 @@ impl Container {
     }
 
     /// Stop and clear all instances
-    pub fn shutdown(&mut self) -> Result<(), HolochainInstanceError> {
-        self.stop_all_instances()?;
-        // @TODO: also stop all interfaces
+    pub fn shutdown(&mut self) {
+        let _ = self
+            .stop_all_instances()
+            .map_err(|error| notify(format!("Error during shutdown: {}", error)));
```

Comment:
Why did we change the this from returning a result? stop_all_instances() seems to be mapping an error

Reply:
Because if one instance errors during shutdown this function would exit early and not reset `self.instances`.

Logically, shutdown should always work. If an instance complains when being shutdown, it should only show in the log.